clusters/ztp-siteconfig/skidoo/skidoo-siteconfig.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "skidoo"
+  namespace: "skidoo"
+spec:
+  baseDomain: "cars.lab"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "img4.9.37-x86-64-appsub"
+# yamllint disable-line rule:line-length
+  sshPublicKey: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAID34lV7EwBk4GDNpnGec0mQdXV3VDLwLiBnvbaikdgLE crucible@oc-client.infra.cars.lab"
+  clusters:
+    - clusterName: "skidoo"
+      clusterLabels:
+        group-vdu: ""
+        common-49: true
+        sites: "r740xls"
+      networkType: "OVNKubernetes"
+      clusterNetwork:
+        - cidr: "10.128.0.0/14"
+          hostPrefix: 23
+      serviceNetwork:
+        - "172.30.0.0/16"
+      machineNetwork:
+        - cidr: "10.60.0.0/24"
+      additionalNTPSources:
+        - 10.40.0.100
+      nodes:
+        - hostName: "super1.skidoo.cars.lab"
+          role: master
+# yamllint disable-line rule:line-length
+          bmcAddress: "idrac-virtualmedia://172.28.11.29/redfish/v1/Systems/System.Embedded.1"
+          bmcCredentialsName:
+            name: "skidoo-bmc-creds-secret"
+          bootMACAddress: "40:A6:B7:3D:B3:70"
+          bootMode: "UEFI"
+          cpuset: "0-3,48-51"
+          nodeNetwork:
+            config:
+              interfaces:
+                - name: enp1s0
+                  type: ethernet
+                  state: up
+                  ipv4:
+                    enabled: true
+                    address:
+                      - ip: 10.60.0.60
+                        prefix-length: 24
+                  ipv6:
+                    enabled: false
```

Comment:
:(

Reply:
This is what @hhamid93 used. See https://github.com/redhat-partner-solutions/vse-carslab-hub/pull/41

Reply:
I will seed a new PR to add ipv6 here.